OpenGL.Platform/SDL2-CS/SDL2_image.cs

```diff
@@ -24,14 +24,6 @@
  * Ethan "flibitijibibo" Lee <flibitijibibo@flibitijibibo.com>
  *
  */
-
-// Copyright (c) 2017 Giawa
```

Comment:
I modified the file to remove unsafe methods.  It was always a hope of mine to avoid unsafe code (or at least exposing unsafe methods) in this OpenGL library.  So I would prefer if we didn't need to use unsafe if possible.  I noticed that `AllowUnsafeBlocks` has also been set back to true for this PR, which would undo that.

Reply:
Ok, I thought it was the same was SDL#, so I just updated it. Sorry.